src/index.js

```diff
@@ -1,9 +1,13 @@
 import Task from './modules/class-task.js';
+import checkItem from './modules/completed.js';
+
 import './style.css';
 
 const btnAdd = document.getElementById('btn-add');
-const taskList = [];
+const btnClearAll = document.querySelector('.clear-btn');
+let taskList = [];
 
+// Event for adding a new task, whenever the button add is cliked
 btnAdd.addEventListener('click', () => {
   const inputTask = document.querySelector('.input-task').value;
   const newTask = new Task(inputTask);
```

Comment:
- [ ] Great job, but I think that something missing here because I am trying to add a new task and I can't see any tasks displayed in the list so kindly Fix that so I can see the tasks also I can't see any tasks added to the local storage🔥🔥💯💯